test/tests/auxkernels/nodal_aux_var/tests

```diff
@@ -1,46 +1,130 @@
 [Tests]
-  issues = '16769b212846275cd8b12e5a89b9e98f3ab7ca83'
-  design = 'syntax/AuxKernels/index.md'
-  [./init_test]
+  issues = '16769b212846275cd8b12e5a89b9e98f3ab7ca83 #22563'
+  design = 'syntax/AuxKernels/index.md Coupleable.md'
+  [init_test]
     type = 'Exodiff'
     input = 'nodal_aux_init_test.i'
     exodiff = 'out_init.e'
-    requirement = "The system shall include the ability to limit the execution of an AuxKernel object to initial setup."
-  [../]
+    requirement = "The system shall include the ability to limit the execution of an AuxKernel "
+                  "object to initial setup."
+  []
 
-  [./test]
+  [test]
     type = 'Exodiff'
     input = 'nodal_aux_var_test.i'
     exodiff = 'out.e'
     requirement = "The system shall include the ability to couple auxiliary variables."
-  [../]
+  []
 
-  [./sort_test]
+  [sort_test]
     type = 'Exodiff'
     input = 'nodal_sort_test.i'
     exodiff = 'nodal_sort_test_out.e'
-    requirement = "The system shall be capable of resolving dependencies between AuxKernel objects regardless of input file order."
-  [../]
+    requirement = "The system shall be capable of resolving dependencies between AuxKernel objects "
+                  "regardless of input file order."
+  []
 
-  [./multi_update_test]
+  [multi_update_test]
     type = 'Exodiff'
     input = 'multi_update_var_test.i'
     exodiff = 'out_multi_var.e'
     issues = '#2099'
-    requirement = "The AuxKernel objects shall be capable of coupling to multiple variables."
-  [../]
+    requirement = "The AuxKernel objects shall be capable of wriyting to to multiple coupled "
+                  "variables."
+  []
+  [multi_update_error]
+    type = RunException
+    input = multi_update_var_error.i
+    expect_err = "writableVariable\(\) can only be called from AuxKernels, ElementUserObjects, or "
+                 "NodalUserObjects\. 'all' is neither of those\."
+    issues = '#22563'
+    allow_deprecated = true
+    requirement = "The writableVariable API shall be limited to use from AuxKernels, "
```

Comment:
We need to generalize these, we shouldn't be referencing any objects/APIs by name

Reply:
I think I made the required changes. Let me know if further edits are needed.